### PR TITLE
Add Closing conditions for pull requests

### PR DESCRIPTION
**Pull request by letsgamedev**
# Description
Habe einige Bedingungen hinzugefügt um das schließen von Pullrequests zu ermöglichen um so einen besseren Überblick gewährleisten zu können.

## Type of change
- [ ] Bug fix (non breaking change that fixes an issue)
- [x] New feature (non breaking change that adds functionality)
- [ ] Tooling (non breaking change that adds functionality to the workflow)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)

### CODE REVIEW

**Comment by b7g** [comment]
It is not stated what happens to pull request older then 30 days that were in draft mode (allowance 60 days) and then reopened (allowance 30 days).
The intent is not to delete them, right?

**Comment by Numenter** [comment]
Maybe look at the last commit date and not at the creation date and just close inactive PRs/drafts

**Comment by b7g** [comment]
I can predict Tom might want to easily see the PRs that can be closed in the overview, without going into every PR.
There is a search filter (the bar on top of the PRs) that can be used for that:

`is:pr is:open updated:<2022-05-30`

this would display only open PRs that had their last commit before the 30 May 2022

**Comment by Numenter** [comment]
This would speed up the merge validation process. #59

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,25 +1,48 @@
...
+
 - **Last commit is 24h or older** and **has more than 10 votes** and **75% positive votes**
 - **Last commit is 72h or older** and **75% positive votes**
+
+## Close pull request without merging conditions
+
+If one of them is true, the pull request is considered as blocked and will be removed/closed
+
+- If a pull request is older than 30 days and not in draft mode
+- If a pull request is older than 60 days
```
Comment (Numenter) [inline CONTRIBUTING.md]:
```suggestion
If its true, the pull request / draft is considered as blocked and will be closed

- **Last commit is 20 days or older**
```

**Review by Numenter — CHANGES_REQUESTED** [review]
Inactive and old are not the same. I would only close inactive PRs and lower the days to 20.
Maybe even lower. It can always be opened again.
#59 will make it easy to sort out inactive PRs.

**Comment by RedstoneMedia** [comment]
Maybe adding a clause that automatically closes PRs if they reach a certain dislike ratio (Maybe 80% ?) after a shorter number of days (Maybe 3 ?) would also be a good idea.

**Comment by Numenter** [comment]
Folks not only give dislikes because they don't like the PR. They dislike if the PR has flaws that they want to be fixed.
Its on the creator to consider, if he wants to edit the PR or to close it.
The forced closing is just to clean up after  abandoned PRs.
_Thats how i see it :P._

**Comment by letsgamedev** [comment]
Ich habe den Vorschlag von Numenter angenommen dass nun der last commit entscheidend ist für das Schließungskriterium. Sollte es also ein PR nicht schaffen innerhalb von 20 Tagen (nach seinem letzten commit, achtung die Gefahr des künstlichen "Am Leben Haltens" ist gegeben durch Mini-commits) durchzusetzen, der PR geschlossen wird.

**Pull request merged**